Project-Codewars-Leaderboard/index.test.mjs

```diff
@@ -18,10 +18,13 @@ test("mocks a fetch function", async () => {
   // functions.
   // In this example, we set up nock so that it looks for GET requests to
   // https://example.com/test (no other URLs will work) and responds with a 200
-  // HTTP status code.
-  const scope = nock("https://example.com").get("/test").reply(200);
+  // HTTP status code, and the body {"user": "someone"}.
+  const scope = nock("https://example.com").get("/test").reply(200, "{\"user\": \"someone\"}");
```

Comment:
Perhaps let's avoid the pain of manually escaping the string?

```suggestion
  const scope = nock("https://example.com").get("/test").reply(200, JSON.stringify({ user: "someone" }));
```

But.. I'm not sure if they'll have come across `JSON.stringify` yet? Does it matter if they haven't?

Reply:
Works for me!